src/python/clawutil/util.py

```diff
@@ -0,0 +1,44 @@
+r"""
+clawutil.util Module  `$CLAW/clawutil/src/python/clawutil/util.py`
+
+Provides general utility functions.
+
+:Functions:
+
+ - fullpath_import: import a module using its full path
+
+"""
+
+def fullpath_import(fullpath, verbose=True):
+    """
+    Return a module imported from a full path name, e.g. if you have
+    a personal enhanced version of the geoclaw topotools module at
+    /full/path/to/topotools.py then instead of:
+
+        from clawpack.geoclaw import topotools
+
+    use:
+
+        from clawpack.clawutil.util import fullpath_import
+        topotools = fullpath_import('/full/path/to/topotools.py')
+
+    Relative imports also work, e.g. 
+
+        topotools = fullpath_import('../topotools.py')
+        
+    To reload the module if you make changes to it, use this function again
+    (rather than using importlib.reload).
+    """
+
+    import os, sys, importlib
+    fname = os.path.split(fullpath)[1]
+    modname = os.path.splitext(fname)[0]
```

Comment:
May want to provide a way to provide `modname`, say if you wanted to keep `topotools` around, but still import something with the same name (kind of an equivalent to `import my_module as my_better_module`).

Reply:
You can do e.g.:
```
my_better_topotools = fullpath_import('/full/path/to/topotools.py')
```
to give it a different name.